src/odemis/gui/cont/multi_point_correlation.py

```diff
@@ -0,0 +1,762 @@
+# -*- coding: utf-8 -*-
+"""
+@author Karishma Kumar
+
+Copyright © 2025, Delmic
+
+Handles the controls for performing correlation using 3DCT algorithm.
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+import itertools
+import logging
+import os
+import queue
+import re
+import threading
+from enum import Enum
+
+import numpy
+import wx
+# IMPORTANT: wx.html needs to be imported for the HTMLWindow defined in the XRC
+# file to be correctly identified. See: http://trac.wxwidgets.org/ticket/3626
+# This is not related to any particular wxPython version and is most likely permanent.
+import wx.html
+
+from odemis import model
+from odemis.acq.align.tdct import get_optimized_z_gauss, _convert_das_to_numpy_stack, run_tdct_correlation
+from odemis.acq.feature import save_features, FIBFMCorrelationData, Target
+from odemis.acq.stream import StaticFluoStream, StaticSEMStream, StaticStream, StaticFIBStream
+from odemis.gui import conf
+from odemis.gui.util import call_in_wx_main, wxlimit_invocation
+from odemis.model import ListVA
+from odemis.util.dataio import data_to_static_streams
+from odemis.util.interpolation import interpolate_z_stack
+from odemis.util.units import readable_str
+
+
+# create an enum with column labels and position
+class GridColumns(Enum):
+    Type = 0  # Column for "type"
+    X = 1  # Column for "x"
+    Y = 2  # Column for "y"
+    Z = 3  # Column for "z"
```

Comment:
Is there a need for unit in the column name?

Reply:
It can be clear to specify the units